gpkit/keydict.py

```diff
@@ -44,6 +44,8 @@ def __init__(self, *args, **kwargs):
     def update(self, *args, **kwargs):
         "Iterates through the dictionary created by args and kwargs"
         for k, v in dict(*args, **kwargs).items():
+            if hasattr(v, "copy"):
```

Comment:
Yeah. Right now it's really only for numpy arrays, but I figured I'd be a bit more general.